public/manifest.json

```
@@ -1,6 +1,6 @@
 {
-  "short_name": "SushiSwap",
-  "name": "Farm SUSHI with Uniswap LP Tokens",
+  "short_name": "HoneySwap",
```

Should this be `Honeycomb` ?
```suggestion
  "short_name": "Honeycomb",
```